docs/howto/handling_connections.ipynb

```diff
@@ -4,16 +4,176 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# How to handle connections to devices\n",
+    "# How to handle connections to devices"
```

Comment:
This should have a dedicated howto.

Reply:
Why? All of this refers to connection handling.

Reply:
Because it's a different and more advanced use case. How tos are supposed to be short, concise and target a specific question and that how to already covers too many things. So this should be something like "Handling multiple connections with the same backend to the same device"